### PR TITLE
Fixed no records issue in ListView and a style issue

### PR DESCRIPTION
**Pull request by vishalshrm539**
* Fixed no records issue in ListView(raised by LeasePlan).
* Please refer BUG-764865 for more details about the issue.

### CODE REVIEW

File: src/components/templates/ListView/index.tsx [hunk 1]
```diff
@@ -482,7 +485,7 @@ export default function ListView(props) {
     return PCore.getDataPageUtils().getDataAsync(
       referenceList,
       context,
-      payload && payload.dataViewParameters,
+      payload ? payload.dataViewParameters : dataViewParameters,
```
[the remark this answers is not in the document]
Comment (JEAxero) [reply src/components/templates/ListView/index.tsx:488]: Ok. Thanks. In that case, let's put a comment in around line 95 saying that we're using the variable to prepare fr possible upcoming use of a utility function.

**Review by JEAxero — CHANGES_REQUESTED** [review]
Ok with me but please think about and address Siva's question to decide which way to do with that.

File: src/components/templates/ListView/index.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (vishalshrm539) [reply src/components/templates/ListView/index.tsx:489]: > Ok. Thanks. In that case, let's put a comment in around line 95 saying that we're using the variable to prepare fr possible upcoming use of a utility function.

Added that.

**Review by JEAxero — APPROVED** [review]
Tested and looks good.

**Pull request merged**